<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {background-color: rgb(0, 255, 255);
font: 14pt &quot;Arial&quot;;
border-radius: 10px;
border-width: 2px;
hover: rgb(212, 255, 252)
}

QPushButton:hover {
background-color: rgb(100, 255, 255);
}

QPushButton:pressed {
background-color: rgb(27, 198, 186)
}

QLabel{
font: 14pt &quot;Arial&quot;;
}

#logout_button {background-color: rgb(255, 80, 80);
font: 14pt &quot;Arial&quot;;
border-radius: 10px;
border-width: 2px;
hover: rgb(212, 255, 252)
}

#logout_button:hover {
background-color: rgb(255, 136, 136);
}

#logout_button:pressed {
background-color: rgb(186, 3, 3)
}

#welcome_label_analista_calidad{
font: 18pt &quot;Arial&quot;;
}
</string>
  </property>
  <widget class="QLabel" name="labelSupervisor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>BIENVENID@</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TERMINAR SESION</string>
   </property>
  </widget>
  <widget class="QLabel" name="LOGO">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../media/imagenes/logo.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="retro">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RETROALIMENTACIÓN</string>
   </property>
  </widget>
  <widget class="QLabel" name="desglose">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>DESEMPEÑO GENERAL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openDash">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>VER TABLERO</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
